eng/Packages.Data.props

```diff
@@ -206,7 +206,8 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(IsGeneratorLibrary)' == 'true'">
-    <PackageReference Update="Microsoft.TypeSpec.Generator.ClientModel" Version="1.0.0-alpha.20250307.1" />
+    <PackageReference Update="Microsoft.TypeSpec.Generator.ClientModel" Version="1.0.0-alpha.20250317.2" />
+    <PackageReference Update="System.ClientModel" Version="1.3.0" />
```

Comment:
I noticed that the repo-wide version is still 1.2.1. Any reason to not bump that to latest @jsquire?